pkg/render/policyrecommendation.go

```diff
@@ -150,18 +148,11 @@ func (pr *policyRecommendationComponent) clusterRole() client.Object {
 			Verbs:     []string{"get", "list", "watch"},
 		},
 		{
-			APIGroups: []string{"crd.projectcalico.org"},
```

Comment:
ack sorry my fault on my comment being unclean - was referring to changing the APIGroup to be "projectcalico.org" instead of "crd.projectcalico.org"